.jules/runs/cartographer_roadmap_design/pr_body.md

```
@@ -0,0 +1,51 @@
+## 💡 Summary
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)